Action.c

```diff
@@ -407,6 +408,51 @@ static Htop_Reaction actionSetAffinity(State* st) {
 
 }
 
+#ifdef SCHEDULER_SUPPORT
+static Htop_Reaction actionSetSchedPolicy(State* st) {
+   if (Settings_isReadonly())
+      return HTOP_KEEP_FOLLOWING;
+
+   static int preSelectedPolicy = SCHEDULINGPANEL_INITSELECTEDPOLICY;
+   static int preSelectedPriority = SCHEDULINGPANEL_INITSELECTEDPRIORITY;
+
+   Panel* schedPanel = Scheduling_newPolicyPanel(preSelectedPolicy);
+
+   const ListItem* policy;
+   for(;;) {
+      policy = (const ListItem*) Action_pickFromVector(st, schedPanel, 18, true);
+
+      if (policy && policy->key == -1)
+         Scheduling_togglePolicyPanelResetOnFork(schedPanel);
+      else
+         break;
```

Comment:
Slightly more readable if you write like this:
```suggestion
      if (!policy || policy->key != -1)
         break;
      Scheduling_togglePolicyPanelResetOnFork(schedPanel);
```